saleor/account/utils.py

```diff
@@ -101,3 +104,20 @@ def get_random_avatar():
     avatar_name = random.choice(os.listdir(AVATARS_PATH))
     avatar_path = os.path.join(AVATARS_PATH, avatar_name)
     return File(open(avatar_path, "rb"), name=avatar_name)
+
+
+def remove_staff_member(staff):
+    """Remove staff member account only if it has no orders placed.
+
+    Otherwise, switches is_staff status to False.
+    """
+    if staff.orders.exists():
+        staff.is_staff = False
+        staff.user_permissions.clear()
+        staff.save()
+    else:
+        staff.delete()
+
+
+def staff_member_required(f):
```

Comment:
Maybe for better readability, we should introduce `account/decorators` file?

Reply:
Probably it is, but it's used only in Django views. As far as I know, we will be removing all views soon. So I'm not sure if we care. 🤔 
What do you think @maarcingebala, @NyanKiyoshi?

Reply:
I wouldn't bother with that, soon it won't be needed at all.

Reply:
So if this will be not needed soon then maybe it is a good moment to put such logic in one place? Dropping the whole file of old methods/functions will be faster (and I assume safer) than checking each function one by one. We had a similar solution for old mutations and queries. We just moved them to the `deprecated_` files. It was easier to remove deprecated mutations/queries when we had them in one place, instead of searching in different files
@maarcingebala @IKarbowiak ?

Reply:
I would first merge this PR, then drop Storefront 1.0 and we'll see what needs to be moved to a seaprate file.